app/components/Admin/ProposedService.js

```diff
@@ -86,6 +95,16 @@ class ProposedService extends React.Component {
     }
     return scheduleOutput;
   }
+
+  renderCategoryFields() {
+    return (
+      <div key={"cat"} className="request-entry">
+        <p className="request-cell name">Categories</p>
+        <textarea readOnly className="request-cell value" value={this.state.categories} />
```

Comment:
Yeah, after talking with joe we realized it was complicated. All of the other fields are string values, but here, you want to edit the object. As an enhancement, I can write the stuff to map the edited strings to the category objects.